releasing/plugin-commands-publishing/src/pack.ts

```diff
@@ -127,6 +127,21 @@ async function readReadmeFile (filesMap: Record<string, string>) {
   return readmeFile
 }
 
+function getGzipConfig () {
+  const gzipConfig: ZlibOptions = {}
+
+  const level = process.env.PNPM_PACK_GZIP_LEVEL
```

Comment:
This should be a new setting in `@pnpm/config` and passed through options.

Reply:
Just figured out how `@pnpm/config` works when I saw your commit.

Also thanks for adding the changelog entry, completely forgot about that!